src/api.v2/controllers/experimentController.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/prefer-default-export */
+const config = require('../../config');
+
+const experiment = require('../model/experiment');
+const userAccess = require('../model/userAccess');
+
+const AccessRole = require('../../utils/enums/AccessRole');
+
+const { OK } = require('../../utils/responses');
+
+const createExperiment = async (req, res) => {
+  const { params: { experimentId }, user, body } = req;
+
+  const { name, description } = body;
+
+  await experiment.create({ id: experimentId, name, description });
+
+  await Promise.all([
+    userAccess.create(
+      { user_id: user.sub, experiment_id: experimentId, access_role: AccessRole.OWNER },
+    ),
+    userAccess.create(
+      { user_id: config.adminArn, experiment_id: experimentId, access_role: AccessRole.OWNER },
```

Comment:
why do we need to insert the same user twice? What is `user.sub` equal to?

Reply:
This is copying the behaviour in the v1 variant of `experimentCreation`, one user is the creator of the experiment, the other is the admin

Reply:
user.sub is the id assigned through congito to users